src/integration/HubspotApiModel.ts

```diff
@@ -14,10 +13,14 @@ export class HubspotApiModel {
      * @param objectId objectID of contact to query
      * @param paramToGet query parameters to filter by. eg. 'email'.
      */
-    public async getContact(contactObjectID: number, paramToGet?: string){
-        console.log("this is current user ", get_current_user());
-        let token:any = await PollTokensFromDatabase(get_current_user());
+    public async getContact(contactObjectID: number,hub_id:number, paramToGet?: string){
+
+        console.log("this is current user ", hub_id);
+
+        let token:any = await PollTokensFromDatabase(hub_id.toString());
```

Comment:
We should have this in a try-catch, or at least check to make sure token is defined before line 22. `PollTokensFromDatabase` does not throw an error if no token exists, it simply logs that.